package.json

```diff
@@ -16,6 +16,7 @@
   "dependencies": {
     "antd": "^5.8.4",
     "classnames": "^2.3.2",
+    "immutability-helper": "^3.1.1",
```

Comment:
reason of adding ?

Reply:
This library provide update function which help to update nested object with changing original object as we are changing the layout width, heights top and bottom value when component has been drag